lib/ace/ace.js

```diff
@@ -55,6 +55,7 @@ require("./multi_select");
 require("./mode/folding/fold_mode");
 require("./theme/textmate");
 require("./ext/error_marker");
+require("./lib/katex/katex")
```

Comment:
About KaTeX.
It should be loaded dynamically from the code, like papeeria_latex mode when we call `session.setMode("ace/mode/papeeria_latex");`

`<script>` tag will load KaTeX on page load, `require` call from ACE will load it when ACE loads -- neither is ok.